WooCommerce/Classes/ViewRelated/Orders/Collect Payments/CollectOrderPaymentUseCase.swift

```diff
@@ -549,6 +582,22 @@ private extension CollectOrderPaymentUseCase {
     }
 }
 
+// MARK: Backend receipts presentation
+private extension CollectOrderPaymentUseCase {
+    /// Prepares and presents the backend receipt modally
+    ///
+    func presentBackendReceiptModally(receipt: Receipt, onCompleted: @escaping (() -> Void)) {
+        let receiptViewModel = ReceiptViewModel(receipt: receipt,
+                                                orderID: self.order.orderID,
+                                                siteName: self.stores.sessionManager.defaultSite?.name)
+        let receiptViewController = ReceiptViewController(viewModel: receiptViewModel, onDismiss: {
+            onCompleted()
+        })
+        let navigationController = UINavigationController(rootViewController: receiptViewController)
+        self.rootViewController.present(navigationController, animated: true)
```

Comment:
```suggestion
                                                orderID: order.orderID,
                                                siteName: stores.sessionManager.defaultSite?.name)
        let receiptViewController = ReceiptViewController(viewModel: receiptViewModel, onDismiss: {
            onCompleted()
        })
        let navigationController = UINavigationController(rootViewController: receiptViewController)
        rootViewController.present(navigationController, animated: true)
```

Reply:
Good catch, I forgot to remove them when extracted the method from its previous place 😅 . Fixed: https://github.com/woocommerce/woocommerce-ios/pull/11896/commits/4ce6730d080242280b103f710fc6cd2bbd61b60f